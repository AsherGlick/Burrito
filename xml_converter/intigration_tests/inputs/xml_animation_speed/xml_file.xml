<OverlayData>
    <MarkerCategory DisplayName="My Category" Name="mycategory">
    </MarkerCategory>

    <POIs>
        <Trail AnimSpeed="0" Type="mycategory" />
        <Trail AnimSpeed="1.0" Type="mycategory" />
        <Trail AnimSpeed="3.14" Type="mycategory" />
        <Trail AnimSpeed="123.456" Type="mycategory" />
        <Trail AnimationSpeed="0" Type="mycategory" />
        <Trail AnimationSpeed="1" Type="mycategory" />     
        <Trail AnimationSpeed="3.14" Type="mycategory" />
        <Trail AnimationSpeed="123.456" Type="mycategory" />
        <Trail AnimSpeed="-3.14" Type="mycategory" />
        <Trail AnimSpeed="-123.456" Type="mycategory" />
        <Trail AnimationSpeed="-3.14" Type="mycategory" />
        <Trail AnimationSpeed="-123.456" Type="mycategory" />
    </POIs>
</OverlayData>
