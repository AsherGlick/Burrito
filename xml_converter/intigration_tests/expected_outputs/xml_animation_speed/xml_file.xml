<OverlayData>
<MarkerCategory  DisplayName="My Category" Name="mycategory">
</MarkerCategory>

<POIs>
<Trail  AnimSpeed="0.000000" Type="mycategory"/>
<Trail  AnimSpeed="3.140000" Type="mycategory"/>
<Trail  AnimSpeed="123.456001" Type="mycategory"/>
<Trail  AnimSpeed="0.000000" Type="mycategory"/>
<Trail  AnimSpeed="3.140000" Type="mycategory"/>
<Trail  AnimSpeed="123.456001" Type="mycategory"/>
<Trail  AnimSpeed="-3.140000" Type="mycategory"/>
<Trail  AnimSpeed="-123.456001" Type="mycategory"/>
<Trail  AnimSpeed="-3.140000" Type="mycategory"/>
<Trail  AnimSpeed="-123.456001" Type="mycategory"/>
</POIs>
</OverlayData>
